frontend/src/pages/programs.tsx

```diff
@@ -19,6 +21,10 @@ export default function Programs() {
 
   const [programs, setPrograms] = useState<ProgramMap>({});
 
+  const { firebaseUser, piaUser, loadingUser } = useContext(UserContext);
+  const _loadingUser = loadingUser;
```

Comment:
what do you need this for?

Reply:
I had it originally because I had an error for unused variable, but I don't think I need loadingUser anyways, so I'll remove that.

Reply:
ah gotcha, also I just put up a PR (#94) where I put `isAdmin` into the userContext so you could just grab that here if you want:
`const { isAdmin } = useContext(UserContext);`